app/models/ext_management_system.rb

```diff
@@ -27,7 +27,7 @@ def self.supported_types_and_descriptions_hash
 
   belongs_to :provider
   belongs_to :tenant
-
+  has_many :container_deployments, :inverse_of => :ext_management_system
```

Comment:
I think you'll need two has_many here
```ruby
has_many :container_deployments_on, :inverse_of => : deployed_on_ems
has_many :container_deployments, :inverse_of => : deployed_ems
```

And yes, maybe rails gets which inverse is used, but the doc says it tries hard to do so and I'm not sure when it works and when not, so I'll always add it whenever the keys and classes dont map 100%